db/migrate/20170315221025_edit_user_image_column.rb

```diff
@@ -0,0 +1,6 @@
+class EditUserImageColumn < ActiveRecord::Migration[5.0]
+  def change
+    change_column_default :users, :image, from: nil, to: "https://avatars3.githubusercontent.com/u/1489337?v=3&s=460"
```

Comment:
Nice that you've seen how to do this on the database level. Unfortunately this will also make it rather difficult to change. Consider doing it on the model level instead, using an active record callback.
See http://stackoverflow.com/questions/328525/how-can-i-set-default-values-in-activerecord#5127684